facturacion_mexico/tests/test_pdf_custom_section.py

```
@@ -0,0 +1,198 @@
+def _mock_si(**kwargs):
+	si = MagicMock()
+	si.po_no = kwargs.get("po_no", "")
+	si.remarks = kwargs.get("remarks", "")
+	si.grand_total = kwargs.get("grand_total", 1000.0)
+	si.currency = kwargs.get("currency", "MXN")
+	si.due_date = kwargs.get("due_date", "2026-07-31")
+	si.payment_schedule = kwargs.get("payment_schedule", [])
+	return si
+
+
+def _apply_settings_and_build(si, metodo_pago, company, settings):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_apply_settings_and_build`

(ANN202)